server/src/main.rs

```diff
@@ -1,20 +1,48 @@
 mod collection;
+mod error;
 mod item;
 mod prompt;
 mod service;
 mod service_manager;
 mod session;
 
-use service::{Result, Service};
+use clap::Parser;
+use error::Error;
```

Comment:
```suggestion
use crate::error::Error;
```

Otherwise you are confusing rustfmt, also run the codebase with `cargo +nightly fmt --all`, it will sort and group imports for you.

Reply:
Done, thanks!